# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../snowbox/destdir.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- snowbox-1.0/Makefile.vanilla	2008-07-26 10:09:45.000000000 +0200
+++ snowbox-1.0/Makefile	2008-07-26 10:10:36.000000000 +0200
@@ -9,20 +9,20 @@
 	@echo "Type \"make install\" to install Snowbox."
 
 install:
-	if [ ! -d "$(PREFIX)/sbin" ]; then \
-		mkdir -p $(PREFIX)/sbin; \
+	if [ ! -d "$(DESTDIR)$(PREFIX)/sbin" ]; then \
+		mkdir -p $(DESTDIR)$(PREFIX)/sbin; \
 	fi
-	install snowbox $(PREFIX)/sbin
+	install snowbox $(DESTDIR)$(PREFIX)/sbin
 
-	if [ ! -d "$(PREFIX)/man/man8" ]; then \
-		mkdir -p $(PREFIX)/man/man8; \
+	if [ ! -d "$(DESTDIR)$(PREFIX)/man/man8" ]; then \
+		mkdir -p $(DESTDIR)$(PREFIX)/man/man8; \
 	fi
-	install -m 0644 snowbox.8 $(PREFIX)/man/man8
+	install -m 0644 snowbox.8 $(DESTDIR)$(PREFIX)/man/man8
 	
-	if [ ! -d "/etc/snowbox" ]; then \
-		mkdir -p /etc/snowbox; \
-		install -m 0600 config /etc/snowbox; \
-		install -m 0600 user.auth.example /etc/snowbox; \
+	if [ ! -d "$(DESTDIR)/etc/snowbox" ]; then \
+		mkdir -p $(DESTDIR)/etc/snowbox; \
+		install -m 0600 config $(DESTDIR)/etc/snowbox; \
+		install -m 0600 user.auth.example $(DESTDIR)/etc/snowbox; \
 	fi
 
 dist:
